extensions/wikia/AdEngine/js/provider/directGptMobile.js

```diff
@@ -2,14 +2,17 @@
 define('ext.wikia.adEngine.provider.directGptMobile', [
 	'ext.wikia.adEngine.provider.factory.wikiaGpt',
 	'ext.wikia.adEngine.uapContext',
+	'wikia.document',
 	'wikia.window'
-], function (factory, uapContext, win) {
+], function (factory, uapContext, doc, win) {
 	'use strict';
 
 	// TODO: ADEN-3542
 	function dispatchNoUapEvent(slotName) {
+		var event = doc.createEvent('CustomEvent');
```

Comment:
Does it make sense to create `CustomEvent` even though we may not use it if the condition below isn't met?